src/main/java/com/myFirstProject/myFirstProject/exception/UserHaveNoAccountException.java

```diff
@@ -4,9 +4,9 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 
 @ResponseStatus(HttpStatus.BAD_REQUEST)
-public class UserHaveNoAccount extends ServiceException{
+public class UserHaveNoAccountException extends ServiceException{
```

Comment:
bad class name, even with grammar mistakes: 
UserHasNotAccountException

or if it is from Account logic:
AccountNotBelongToUserException